src/renderer/helpers/utils.js

```diff
@@ -611,7 +611,26 @@ export function getVideoParamsFromUrl(url) {
 
   function extractParams(videoId) {
     paramsObject.videoId = videoId
-    paramsObject.timestamp = urlObject.searchParams.get('t')
+    let timestamp = urlObject.searchParams.get('t')
+    if (timestamp && (timestamp.includes('h') || timestamp.includes('m') || timestamp.includes('s'))) {
```

Comment:
This breaks support for links that don't include a letter after the number of seconds which is something we previously supported. Which means you can't copy a video link with a timestamp from FreeTube's watch page and use it in FreeTube anymore without modifying the URL.

Reply:
I haven't actually tested the code yet, but that if checks for whether a timestamp exists AND that it contains one of those 3 letters, so theoretically at least based on reading the code number only timestamps won't work anymore.

Reply:
It should have the same behavior if it's number only since L614 and L633 are outside of the if statement

Reply:
Ah okay I missed that, sorry.